scale/docs/architecture/triggers.rst

```diff
@@ -95,6 +111,18 @@ A valid ingest trigger rule configuration is a JSON document with the following
         types that are listed here tagged to the file in order to trigger this rule. If not provided, the field defaults
         to [] and no data types are required.
 
+    **any_of_data_types**: JSON array
+    
+        The *any_of_data_types* field is an optional list of data type strings. If an ingested file has **any** tagged 
+        data type that is listed here, the rule will trigger. If not provided, the field defaults to [] and no data 
```

Comment:
I would say "An ingested file must have at least one of the data types that are listed here tagged to the file in order to trigger this rule."